.github/workflows/ci.yml

```diff
@@ -15,6 +15,18 @@ env:
   docker_repository: nlpsandbox/phi-deidentifier-app
 
 jobs:
+  build:
+    env:
+      working-directory: ./client
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install modules
+      working-directory: ${{env.working-directory}}
+      run: npm ci
+    - name: Run ESLint
+      working-directory: ${{env.working-directory}}
+      run: npx eslint
```

Comment:
Consider adding the script `npm run lint` to *package.json* to make it easy to developer to lint the project. The CI/CD workflow should then rely on the command `npm run lint` (requires to install dev npm dependencies with `npm install --only=dev`).